src/lib/jam/rooms.ts

```diff
@@ -0,0 +1,31 @@
+// TODO -- I need to rename this as there is already a jam store file
+// I can merge them but either case, I need to rename these variable names
+import { agent, type JamRoom } from '@lib/api';
+import fuzzysort from 'fuzzysort';
+import { derived, get, writable } from 'svelte/store';
+
+export const jamRoomStore = writable<JamRoom[]>([]);
+
+export const fetchState = writable<'loading' | 'done'>('loading');
+
+export async function getJamRooms() {
+    const rooms = await agent.jams.list();
+    jamRoomStore.set(rooms);
+    fetchState.set('done');
+    return rooms;
+}
```

Comment:
If we're going to update state here, I think we need to remove the update from here :
https://github.com/Rapidmidiex/rmx-web-client/pull/27/files#diff-09a088fcbf6bcdb34471126f2cc170ec44c4a9456d2ca90eccbb51089c64b390R11